src/plugins/GUITestBase/src/tests/common_scenarios/msa_editor/GTTestsMsaEditor.cpp

```diff
@@ -4615,6 +4615,32 @@ GUI_TEST_CLASS_DEFINITION(test_0090) {
     }
 }
 
+GUI_TEST_CLASS_DEFINITION(test_0091) {
+    // 1. Open file _common_data\scenarios\msa\nucl_with_leading_gaps.aln
+    GTFileDialog::openFile(os, testDir + "_common_data/scenarios/msa/nucl_with_leading_gaps.aln");
+    GTUtilsMsaEditor::checkMsaEditorWindowIsActive(os);
+    GTUtils::checkExportServiceIsEnabled(os);
+
+    // 2. Do document context menu {Export->Export aligniment to amino format}
+    // 3. Translate with "Include gaps"
+    GTUtilsDialog::waitForDialog(os, new ExportMSA2MSADialogFiller(os, -1, sandBoxDir + "GUITest_common_scenarios_msa_editor_test_0091.aln", true));
+    GTUtilsDialog::waitForDialog(os, new PopupChooser(os, QStringList() << MSAE_MENU_EXPORT << "amino_translation_of_alignment_rows"));
+    GTWidget::click(os, GTUtilsMdi::activeWindow(os), Qt::RightButton);
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    // 4. Copy to clipboard
+    GTUtilsMSAEditorSequenceArea::selectArea(os);
+    GTUtilsDialog::waitForDialog(os, new PopupChooser(os, QStringList() << "MSAE_MENU_COPY"
+                                                                        << "copy_selection"));
+    GTMouseDriver::click(Qt::RightButton);
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    // Expected: TAVS\nXXVS
+    const QString clipboardText = GTClipboard::sequences(os);
+    const QString expectedMSA = "TAVS\nXXVS";
```

Comment:
Why have you selected 'X' option instead of '-' variant? 
I do not know which is better here and unless you have a strong opinion that 'X' is better than '-' I would leave this decision to a user: ask user if to translate incomplete triplets to 'X' or to '-'.

One of the sources for a 'strong opinion' would be a reference to some existing tool that prefers one over another.

Reply:
I have added a possibility to choose